event-checklist.md

```diff
@@ -0,0 +1,104 @@
+
+You will be provided with access to two systems for financial administration, being;
+* Westpac online banking
+* Xero for accounting and general ledger functions
+
+A Council member or delegate will provide instructions and an overview on how to use the financial systems. In order to gain access to Westpac, you will need to provide the following information to [the Linux Australia Treasurer](mailto:treasurer@linux.org.au);
+
+* Westpac customer number (obtained by providing 100 points of ID at a Westpac branch - passport, driver's licence, Medicare card etc.)
+* Date of Birth
+* Mobile number
+* Full legal name
+* preferred email address
+
+
+It is expected that your event subcommittee will provide budgets for review by the Council, to ensure strong financial oversight. Budgetary guidelines are dealt with in detail in the [event subcommittee policy](subcommittee_policy_v2.md).
+
+It is expected that financial transactions are recorded diligently using the financial systems, including recording of income, outgoings and expenditure, and reimbursements. All reimbursements to subcommittee members should fall within [Linux Australia's expense policy](expense_policy.md). The subcommittee is able to implement their own expense policy if approval for an amended expense policy is sought and granted by Council.
+
+##Event infrastructure
+Once financials are set up, you will need to give consideration to the event infrastructure, such as domain name, event registration system and so on. For larger events, you may wish to use [ZooKeepr](http://zookeepr.org/), custom software written specifically for events such as [linux.conf.au](https://linux.conf.au). Your event may be smaller in scale and you may prefer to have your own domain, and to use an alternative registration gateway. If you need to host a site, you are able to then claim back the costs involved. Alternatively, depending on your needs, the Linux Australia Admin Team may be able to provide hosting infrastructure.
+
+##Event code of conduct
+It is expected that your event adopt a Code of Conduct, similar in scope and nature to the [Linux Australia event Code of Conduct](code_of_conduct.md). If your event is going to adopt a Code of Conduct different to the Linux Australia Code of Conduct, the event Code of Conduct must be approved by Linux Australia Council.
+
+##Insurance
+Linux Australia holds a broad insurance policy covering items such as Volunteers at Linux Australia events and public liability insurance. The event subcommittee must ensure that the coverage held by Linux Australia is suitable for the activities being held under the event. The venue where the event is being held may request a Certificate of Currency of insurance, which the Linux Australia Council can provide.

```

Comment:
Might be worth noting if an event is doing anything out of the ordinary they highlight it to the council.